R/utils-add_p_tests.R

```diff
@@ -508,7 +508,7 @@ add_p_test_ancova_lme4 <- function(data, variable, by, group, conf.level = 0.95,
               is_empty(adj.vars) ~ list("One-way ANOVA with random intercept"),
               TRUE ~ list("ANCOVA with random intercept")
             ),
-          fmt_fun = list(NULL)
+          fmt_fn = list(as.character)
```

Comment:
fmt_fn is default in cards, while fmt_fun is default in gtsummary right? I wanted to check if it was actually aligned but there are many instances (ofc)

Reply:
When I originally named them, I thought they were the same.... 😭 But now wondering if it's too late to change them to match.

Reply:
not in this PR ahah